app/src/main/java/com/kickstarter/viewmodels/RewardViewModel.java

```diff
@@ -210,7 +210,7 @@ public RewardViewModel(final @NonNull Environment environment) {
       .subscribe(usdConversionTextViewIsHidden);
 
     projectAndReward
-      .map(pr -> ksCurrency.format(pr.second.minimum(), pr.first, true, true))
+      .map(pr -> ksCurrency.format((float) Math.ceil(pr.second.minimum()), pr.first, true, true))
```

Comment:
It might be nice if `format` could take a rounding mode option, instead of manually using `ceil` and casting back to a float.